src/internal/clusterstate/v2.8.0_test.go

```diff
@@ -14,6 +17,15 @@ import (
 	"github.com/pachyderm/pachyderm/v2/src/internal/testetcd"
 )
 
+func sortCommits(commitInfos []v2_8_0.CommitInfo) {
```

Comment:
Since we are only using this for comparing commits in tests, check out [`cmpopts.SortSlice`](https://pkg.go.dev/github.com/google/go-cmp/cmp/cmpopts#SortSlices).